roles/update/templates/workload_launch.sh.j2

```diff
@@ -42,8 +42,8 @@ function os_cmd {
 
 function set_vm_ip {
     ## assign floating ip or external ip
-    local workload_sriov={{workload_sriov|default(false) | bool | ternary("True", "")}}
-    if [ -n "${workload_sriov}" ]; then
+    local fixed_ip={{ $1|default(false) | bool }}
```

Comment:
how it should work? Here I see bash + ansible in jinja2 template.

Reply:
+1 this is mixing two worlds,, it is not gonna work

Reply:
ok thanks, will test and update it.